console/src/test/java/org/web3j/console/WalletTester.java

```diff
@@ -23,7 +23,7 @@
 
     static final char[] WALLET_PASSWORD = PASSWORD.toCharArray();
 
-    IODevice console;
+    public IODevice console;
```

Comment:
does this need to be public?

Reply:
no, it doesn't I might have changed it during testing and forgot about it. Thanks for the catch.